.github/workflows/bat.yml

```diff
@@ -55,6 +55,17 @@ jobs:
         uses: ./
         with:
           command: exp = getenv('GITHUB_WORKSPACE'), act = pwd, assert(strcmp(act, exp), strjoin({act exp}, '\n'));
+
+      - name: Run MATLAB statement with arguments
+        uses: ./
+        with:
+          command: disp("Hello world!!")
+          startup-options: -nojvm -nodesktop -logfile mylog.log
+          
+      - name: Validate that command ran with arguments
```

Comment:
1) Not user visible, but why statement and command (two different things)

2) I thought you are validating "that command". Maybe an article after "that" helps clarify.

Reply:
Yeah, how about something along the lines of "Validate that the previous command ran with arguments?"

Reply:
Is fine, shorter options:

Validate that the command ...
Test if command ...